.github/DEVELOPMENT.md

```diff
@@ -43,8 +43,11 @@ much easier and reduces the chance of introducing unintended changes in behavior
 
 We recommend you use IntelliJ as your IDE. The code style template for the
 project can be found in the [codestyle](https://github.com/airlift/codestyle)
-repository along with our general programming and Java guidelines. In addition
-to those you should also adhere to the following:
+repository along with our general programming and Java guidelines. 
+
+To run checkstyle checks before opening a PR: `./mvnw checkstyle:check@checkstyle`
```

Comment:
Does `./mvnw validate` seem to do what you want? It'll catch more things (if the goal is to make sure people submit "working" PRs).

Reply:
Oh that is even better since it runs checkstyle + dep enforcer and other things 👍